### DISCUSSION TITLE
New mastodon instance seems isolated from the rest of the world

### DISCUSSION THREAD

**Discussion by nvsajeeva** [mastodon/mastodon#21127] [Q&A]
did setup a new mastodon instance by following their official [documentation](https://docs.joinmastodon.org/admin/prerequisites/). Cloud provider: AWS OS: Ubuntu 20.04 mastodon version: tag v3.5.5

Everything looks good. I can sign up and publish toots. However, the instance seems isolated from the rest of the world. I do not get any results when I search for something in the search text area.

Also, I don't see anything when I click on "Home", "Notifications", "Explore", "Local", "Federated" tabs except for my own toots.

I checked the logs for each events I have mentioned above, but couldn't find any errors

```
Nov 19 18:59:42 mastodon bundle[962]: [2f300833-5315-46c0-a896-476795ededd3] method=GET path=/api/v1/announcements format=html controller=Api::V1::AnnouncementsController action=index status=200 duration=8.95 view=0.81 db=2.90
Nov 19 18:59:54 mastodon node[904]: verb f63957b3-a3a1-4c02-bd4e-effab9a79982 Starting stream from timeline:public:local for 109367375149862625
Nov 19 18:59:54 mastodon node[904]: verb Subscribe timeline:public:local
Nov 19 18:59:54 mastodon bundle[960]: [ba690cc1-f2f9-46dd-a566-914e7125f875] method=GET path=/api/v1/timelines/public format=html controller=Api::V1::Timelines::PublicController action=show status=200 duration=26.52 view=5.64 db=4.51
Nov 19 18:59:56 mastodon node[904]: verb f63957b3-a3a1-4c02-bd4e-effab9a79982 Ending stream from timeline:public:local for 109367375149862625
Nov 19 18:59:56 mastodon node[904]: verb Unsubscribe timeline:public:local
Nov 19 18:59:56 mastodon node[904]: verb f63957b3-a3a1-4c02-bd4e-effab9a79982 Starting stream from timeline:public:remote:media for 109367375149862625
Nov 19 18:59:56 mastodon node[904]: verb Subscribe timeline:public:remote:media
Nov 19 18:59:56 mastodon bundle[962]: [258f251b-74b1-4108-8df4-1ac83046f829] method=GET path=/api/v1/timelines/public format=html controller=Api::V1::Timelines::PublicController action=show status=200 duration=14.09 view=0.38 db=2.86
Nov 19 19:00:00 mastodon bundle[519]: 2022-11-19T19:00:00.102Z pid=519 tid=2aeb INFO: queueing Scheduler::InstanceRefreshScheduler (instance_refresh_scheduler)
Nov 19 19:00:00 mastodon bundle[519]: 2022-11-19T19:00:00.103Z pid=519 tid=2ajv class=Scheduler::InstanceRefreshScheduler jid=de4112a35ea1c3ba20f43230 INFO: start
Nov 19 19:00:00 mastodon bundle[519]: 2022-11-19T19:00:00.182Z pid=519 tid=2ajv class=Scheduler::InstanceRefreshScheduler jid=de4112a35ea1c3ba20f43230 elapsed=0.079 INFO: done
```
![image](https://user-images.githubusercontent.com/22984158/202868362-6b0a0442-755f-4eb5-9091-705228926232.png)
![image](https://user-images.githubusercontent.com/22984158/202868366-2f3be289-cade-49bd-b6b1-c4601ee63f6a.png)
![image](https://user-images.githubusercontent.com/22984158/202868367-78263d66-f392-4a9d-bbf9-dfdb3c5695b2.png)
![image](https://user-images.githubusercontent.com/22984158/202868372-bcb9e8ae-922f-42b1-b81d-0dd0d90c1a3f.png)

What am I missing?

**Comment by mjankowski** [comment]
On the third anniversary of this discussion, let me tell you: you need to follow something.

By default, the server only knows about itself and doesn't connect to anything. If you start following things from other servers (or other people join your server and do so) you'll start seeing things.
